anime_downloader/extractors/hydrax.py

```diff
@@ -9,14 +9,41 @@
 class Hydrax(BaseExtractor):
```

Comment:
Working hydrax extractor found here: https://github.com/vn-ki/anime-downloader/blob/08bff87500c140a0e257c9c5bf4b9342fe13e2aa/anime_downloader/extractors/hydrax.py

Reply:
OK nice, but i want to see for qualty also besause you have HD en SD or sothnig like this

Reply:
maybe in the config, i could modify like this (add extractorVersion) ? : 
"siteconfig": {
    'animesvostfr': {
            "servers": [
                "youtubedownloader",
                "gcloud",
                "feurl",
                "fembed",
                "hydrax"
            ]
            "extractorVersion": {
                "hydrax": 2
                "gcloud": 2
            }
        },

Reply:
why should there be versions in the first place? It's the same URLs? If it's because some attributes, those should be handled on a higher level (anime.py)

Reply:
Maybe the First reason is the code in master branch don't work ? (for hydrax)
And i don't want to push a new version like your version on the fantom branch ... 
because i don't want test all sites who use this extractor ?

Reply:
for Gloud 
For example is not logique to use sort_sources and force gcloud.com in the extractor when the base sever in the url is youtubedownloader, feurl or an other think
So i can use a V2 of gcloud only for my french sites and other sites continue to run like before me

Reply:
You don't need to test it for other sites, as the urls passed to them is the same, it doesn't matter if hydrax.net/video123.mp4 comes from animeout or dubbedanime. The reason it doesn't work in master branch is because it's outdated, which will get fixed once kissanime is merged. 

TL;DR extractors are (pretty much) universal

Reply:
> for Gloud
> For example is not logique to use sort_sources and force gcloud.com in the extractor when the base sever in the url is youtubedownloader, feurl or an other think

Yes, feurl should ideally be another extractor, but currently isn't due to pure laziness (one line fix). If it's another link it should be another extractor. 

> So i can use a V2 of gcloud only for my french sites and other sites continue to run like before me

Making a special case for one site with custom provider independent code (not handled by anime.py) will have a really hard time getting past @vn-ki

Reply:
Yes sure but is not the case for gcloud servers. And i need to use fallback quality servers  ;)

> Making a special case for one site with custom provider independent code (not handled by anime.py) will have a really hard time getting past @vn-ki 

No problem you could use my gcloud code in any case: p but it's my first contrib. I don't want to start breaking everything and impose my choices already lol

Reply:
> i need to use fallback quality servers

What do you mean by this? Quality selection is already available in gcloud

Reply:
No, my default quality is 360p and gcloud don't work with fallback qualities when her is not found. Don't work also with the fallback servers order because force to gcloud.com when i choose gloud.net or fleur.com ... or another things.

But another thing is possible if @vn-ki don't want to improve the extractors system. The solution is that i don't use extractors like the first version and make all under no-extractor.

For hydrax i have it :
`{"status":true,"url":"3A0Mmx5bG1vMi5tb25zdGVyM","sources":["sd","hd"],"isCdn":false,"isHls":false,"ads":"https://unreshiramor.com/4/3239564/"}`

i want to choose sd or hd quality of the video.

if self.quality is "1024p" or "720p" -> HD
else -> SD

Do you know how to choose that ?
How to get the HD url or the SD url ?

Reply:
Oh thanks, I did some investigation and it turned out it gave incorrect "file not found" too much. [Update](https://github.com/vn-ki/anime-downloader/commit/f765992ea9ee7ce5e08415b3559696294089d18b) and try now with 
`anime dl "neverland" --provider animevibe -q 720p` and
`anime dl "neverland" --provider animevibe -q 1080p`
(and change gcloud to the primary provider in config)
Notice how it downloads different videos (with according quality)
For proof of fallback try naruto with 1080p
![image](https://user-images.githubusercontent.com/46196380/84591876-515e8180-ae31-11ea-9c49-1c195929bec8.png)
as for hydrax, I'm not sure how the quality is selected, as it returns only one link. From looking at network requests it looks like, www.123.monster is HD and 123.monster is SD, will investigate further.
![image](https://user-images.githubusercontent.com/46196380/84592066-46582100-ae32-11ea-8c55-1bbcc845d7b9.png)

Reply:
i improved your hydrax.py: https://github.com/BlackDady/anime-downloader/blob/master/anime_downloader/extractors/hydrax.py

For HD version you need to add www at the begining